src/System.Private.Interop/src/InteropExtensions/PInvokeMarshal.cs

```diff
@@ -2,35 +2,35 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Debug = System.Diagnostics.Debug;
 using System.Runtime.CompilerServices;
 
+#if BIT64
```

Comment:
This means that you will need to build System.Private.Interop as platform specific. I do not think you want to deal with the complexity that comes with it.

Reply:
Remove them